.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 🔧 Task
+about: Issue for project support and quality assurance
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+
+The Task issue template is for project support and quality assurance items.
+For example, tests or ci related issues, readme update, technical debt, etc. can be created with this template.
+
+For a new feature or an enhancement, please submit a feature request or user story issue.
+
+Thanks for understanding, and for contributing to the project!
+
+-->
+
+**Which area/kind this issue is related to?**
+
+> Uncomment appropriate areas, and delete the rest.
+> For example, `> /area api` would simply become: `/area api`
+
+> /area ci
+> /area api
+> /area library
+> /area registry
+> /area devworkspace
+> /area integration-tests
+> /area test-automation
+> /area releng
+
+
+## Issue Description
+<!--
+    A clear and concise description of what the task is.
+-->
+
+> Please uncommnet and provide the target Odo version if this task is for QE integration test. 
+> Leave if blank if it's a future QE issue.
+> Delete this comment if it's not related QE intergation test.
```

Comment:
There's a typo on `intergation`, should be `integration`

Reply:
updated